lib/src/participant/participant.dart

```diff
@@ -272,6 +267,35 @@ abstract class Participant<T extends TrackPublication>
                 e.kind == lk_models.TrackType.AUDIO));
   }
 
+  // Must be implemented by subclasses.
+  Future<void> unpublishTrack(String trackSid, {bool notify = true});
```

Comment:
should this be moved to local participant? what are the benefits of keeping this in the super class?

Reply:
yeah, moved unpublishTrack, unpublishAllTracks to local participant.